docs/user/installation_and_configuration.rst

```diff
@@ -0,0 +1,368 @@
+following commands: 
+
+.. code-block:: bash
+
+    $ bin/install-jai
+    $ bin/install-jline
+    $ bin/install-vecmath
+
+Finally, test your installation by editing the ``bin/test-geomesa`` file with configuration
+data specific to your setup and running it: 
+
+.. code-block:: bash
+
+    $ bin/test-geomesa
+
+Test the GeoMesa Tools:
+
+.. code-block:: bash
+
+    $ geomesa
+    Using GEOMESA_HOME = /path/to/geomesa-$VERSION
+    Usage: geomesa [command] [command options]
+      Commands:
+        create           Create a feature definition in a GeoMesa catalog
+        deletecatalog    Delete a GeoMesa catalog completely (and all features in it)
+        deleteraster     Delete a GeoMesa Raster Table
+        describe         Describe the attributes of a given feature in GeoMesa
+        explain          Explain how a GeoMesa query will be executed
+        export           Export a GeoMesa feature
+        getsft           Get the SimpleFeatureType of a feature
+        help             Show help
+        ingest           Ingest a file of various formats into GeoMesa
+        ingestraster     Ingest a raster file or raster files in a directory into GeoMesa
+        list             List GeoMesa features for a given catalog
+        querystats       Export queries and statistics about the last X number of queries to a CSV file.
+        removeschema     Remove a schema and associated features from a GeoMesa catalog
+        tableconf        Perform table configuration operations
+        version          GeoMesa Version
+
+
+GeoMesa Tools comes bundled by default with an SLF4J implementation that is installed to the ``$GEOMESA_HOME/lib`` directory
+named ``slf4j-log4j12-1.7.5.jar``. If you already have an SLF4J implementation installed on your Java classpath you may
+see errors at runtime and will have to exclude one of the JARs. This can be done by simply renaming the bundled
+``slf4j-log4j12-1.7.5.jar`` file to ``slf4j-log4j12-1.7.5.jar.exclude``.
+ 
+Note that if no slf4j implementation is installed you will see this error:
+
+.. code::
+
+    SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
+    SLF4J: Defaulting to no-operation (NOP) logger implementation
+    SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
+
+In this case you may download SLF4J from http://www.slf4j.org/download.html. Extract 
+``slf4j-log4j12-1.7.7.jar`` and place it in the ``lib`` directory of the binary distribution. 
+If this conflicts with another SLF4J implementation, it may need to be removed from the lib directory.
+
+
+Installing the Accumulo distributed runtime library
+---------------------------------------------------
+
+The ``geomesa-$VERSION/dist/accumulo`` directory contains the distributed
+runtime jar that should be copied into the ``$ACCUMULO_HOME/lib/ext`` folder on
+each tablet server. This jar contains the GeoMesa Accumulo iterators that are
+necessary to query GeoMesa.
```

Comment:
Updated. However, is it possible that the new procedure is too complex for end users? I could certainly see why GeoMesa developers would want to do the namespace thing, but if I just wanted to get GeoMesa running I would much rather want to do the 1.5 procedure rather than all this mucking about with namespaces.